packages/xstate-store/src/atom.ts

```diff
@@ -68,10 +68,24 @@ export function createAsyncAtom<T>(
   const atom = createAtom<AsyncAtomState<T>>(() => {
     getValue().then(
       (data) => {
-        atom._update({ status: 'done', data });
+        if (atom._update({ status: 'done', data })) {
```

Comment:
isn't this now the same as `.set`? why are we not using `set` here? 🤔

Reply:
I have a future refactor planned for atoms so that _all_ atoms have something like `.receive(…)`, and the "atom logic" determines what to do with a received value